dragula.js

```diff
@@ -82,8 +84,13 @@ function dragula (initialContainers, options) {
     var offset = getOffset(_item);
     _offsetX = getCoord('pageX', e) - offset.left;
     _offsetY = getCoord('pageY', e) - offset.top;
-    renderMirrorImage();
-    drag(e);
+
+    // delay renderMirrorImage to fire click event when mouseup quickly (defaluts 100ms).
+    _timerClick = setTimeout(function(){
```

Comment:
- Name the function
- Extract the function
- Only use `setTimeout` if `o.delay` is greater than `0`
- Otherwise, execute the function immediately